src/module/item/item-sheet.ts

```diff
@@ -166,6 +157,59 @@ export class LancerItemSheet<T extends LancerItemType> extends ItemSheet<ItemShe
     activate_action_editor(html, getfunc, commitfunc);
   }
 
+  _activatePlusMinusButtons(html: any) {
+    const mod_handler = (delta: number) => async (ev: Event) => {
+      if (!ev.currentTarget) return; // No target, let other handlers take care of it.
+      const button = $(ev.currentTarget as HTMLElement);
+      const writeback_parents = button.parents("div[data-writeback_path]");
+      if (writeback_parents.length > 0) {
+        const params = writeback_parents[0].dataset;
+        this._updateCounterData(params.path, params.writeback_path, delta);
+      } else {
+        mod_shared_handler(button, delta);
+        this.submit({});
+      }
+    };
+
+    // Behavior is identical, just +1 or -1 depending on button
+    let decr = html.find('button[class*="mod-minus-button"]');
+    decr.on("click", mod_handler(-1));
+    let incr = html.find('button[class*="mod-plus-button"]');
+    incr.on("click", mod_handler(+1));
+  }
+
+  async _activateCounterListeners(html: JQuery) {
```

Comment:
We're duplicating this handler-setting code on actor and item sheets now, but I don't know where to put them instead. Do we need a helper file for counters?

Reply:
The helpers for generating counter HTML are in `helpers/item.ts`, you can put these functions there. To me the logical place would be around line 1266, after the counter HTML generators.

Reply:
Thanks! I did just that.